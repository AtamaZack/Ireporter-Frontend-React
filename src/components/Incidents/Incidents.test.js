import React from 'react';
import { shallow } from 'enzyme';
import { AllRedflags , mapStateToProps} from './index';

describe('<AllRedflags />', () => {
  let wrapper;
  const props = {
    getAllRedflags: jest.fn()
  };
  beforeEach(()=>{
    wrapper = shallow(<AllRedflags {...props}/>);
  })

  it('should render the all redflags component', () => {  
    expect(wrapper).toMatchSnapshot();
  }); 



  it('mapStateToProps', ()=>{
    const initialState = { redflagsReducer:{redflags: []}
    };
    expect(mapStateToProps(initialState)).toEqual({redflags: []})

  });
});

