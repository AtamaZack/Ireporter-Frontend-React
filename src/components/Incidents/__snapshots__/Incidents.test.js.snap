// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AllRedflags /> should render the all redflags component 1`] = `ShallowWrapper {}`;
